packages/venia-concept/src/components/Checkout/Receipt/__tests__/receiptContainer.spec.js

```diff
@@ -0,0 +1,54 @@
+import actions from 'src/actions/checkoutReceipt';
+import { continueShopping, createAccount } from 'src/actions/checkout';
+import Container from '../receiptContainer';
+import Receipt from '../receipt';
+
+jest.mock('src/drivers', () => ({
+    connect: jest.fn((mapStateToProps, mapDispatchToProps) =>
+        jest.fn(component => ({
+            component,
+            mapStateToProps,
+            mapDispatchToProps
+        }))
+    )
+}));
```

Comment:
I wonder if we shouldn't provide this as a utility, seems like a lot of test files will need it?

Reply:
Yea I think we could -- I just didn't do that in this PR.